source/features/batch-open-conversations.tsx

```diff
@@ -63,12 +63,49 @@ async function init(): Promise<void | false> {
 				Open selected
 			</button>
 		);
+	} else {
+		const issuesToolbar = select('#js-issues-toolbar')!;
+		issuesToolbar.prepend(
+			<div className="mr-3 d-none d-md-block">
+				<input type="checkbox" data-check-all="" aria-label="Select all issues" autoComplete="off"/>
+			</div>
+		);
+		issuesToolbar.append(
+			<div className="table-list-triage flex-auto js-issues-toolbar-triage">
+				<span className="text-gray table-list-header-toggle">
+					<span data-check-all-count="">1</span> selected
+					{' '}
```

Comment:
```suggestion
					<span data-check-all-count="">1</span>
					{' selected '}
```

Reply:
ok - i had this version before, but the eslint example suggested more this empty string and in codebase it was already one `{' '}`

Reply:
i will change it